packages/rancher-monitoring/generated-changes/patch/charts/k3sServer/templates/pushprox-servicemonitor.yaml.patch

```diff
@@ -0,0 +1,54 @@
+--- charts-original/charts/k3sServer/templates/pushprox-servicemonitor.yaml
++++ charts/charts/k3sServer/templates/pushprox-servicemonitor.yaml
+@@ -13,6 +13,51 @@
+     params:
+       _scheme: [https]
+     {{- end }}
++    honorLabels: true
++{{- if .Values.metricRelabelings }}
++    metricRelabelings:
++{{ tpl (toYaml .Values.metricRelabelings | indent 4) . }}
++{{- end }}
++{{- if .Values.relabelings }}
++    relabelings:
++{{ toYaml .Values.relabelings | indent 4 }}
++{{- end }}
```

Comment:
same comments as above; let's avoid patching this subchart if it could be a commonly used feature anyways.